ajax/libs/event-drops/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "event-drops",
+  "filename": "eventDrops.min.js",
```

Comment:
This should be `eventDrops.js`, not `eventDrops.min.js`, as it's already minified.